src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/model/MacPackage.java

```diff
@@ -34,7 +35,7 @@ public interface MacPackage extends Package, MacPackageMixin {
     @Override
     default AppImageLayout appImageLayout() {
         if (isRuntimeInstaller()) {
-            return RUNTIME_PACKAGE_LAYOUT;
+            return RuntimeLayout.DEFAULT;
```

Comment:
Why is this change? It looks wrong. The layout of the output bundle should have the "Contents" folder. This is how it was before the change.

Reply:
> RUNTIME_PACKAGE_LAYOUT points to "Contents/Home". Bundle is "Contents/Home", "Contents/MacOS" and "Contents/Info.plist".

Right, so `RUNTIME_PACKAGE_LAYOUT` points to a bundle just like [ApplicationLayoutUtils.MAC_APPLICATION_LAYOUT](https://github.com/openjdk/jdk/blob/5cf672e7784b9a9a82f29977a072b162cc240fd1/src/jdk.jpackage/share/classes/jdk/jpackage/internal/ApplicationLayoutUtils.java#L43) points to a bundle too. If you need "Contents/MacOS" in the `RUNTIME_PACKAGE_LAYOUT`, add it to it.